Source/MLX/Documentation.docc/Organization/arithmetic.md

```diff
@@ -181,6 +182,8 @@ Note: the `-` and `/` operators are not able to be linked here.
 
 - ``MLXArray/matmul(_:stream:)``
 - ``matmul(_:_:stream:)``
+- ``blockMaskedMatmul(_:_:blockSize:maskOut:maskLHS:maskRHS:stream:)``
+- ``addmm(_:_:_:alpha:beta:stream:)``
```

Comment:
I think the inconsistency is not ideal here (and in MLX core). We have `block_masked_mm` and `addmm` which should really be `add_mm` if we were consistent.

Since we have an opportunity to be more rigorous here, how do you feel about:

```
blockMaskedMM
addMM
``

It's more consistent with Python but respects the camel case.

Reply:
Sounds good, I will make the change (same deprecation technique will supply the suggestion for the new spelling)

Reply:
Thanks!